tests/test_routes_agent_auth_requests.py

```
@@ -611,9 +611,322 @@ async def test_approve_canvas_scopes_with_blank_project_is_rejected(
+            load_or_create_signing_keypair,
+        )
+        from tinyagentos.agent_grants_store import AgentGrantsStore
+        from tinyagentos.projects.project_store import ProjectStore
+
+        registry = AgentRegistryStore(tmp_path / "reg-assign.db")
+        await registry.init()
+        grants = AgentGrantsStore(tmp_path / "grants-assign.db")
+        await grants.init()
+        pstore = ProjectStore(tmp_path / "projects-assign.db")
+        await pstore.init()
+        priv, pub = load_or_create_signing_keypair(tmp_path / "keys-assign")
```

Ruff (0.15.21)

[warning] Unpacked variable `priv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)